### PR TITLE
fix(cloud.logs): order button

### PR DESCRIPTION
**Pull request by pdepaepe**
Hello,

This PR will fix following elements:

- Point LDP order button on server section to the right place

Thanks for your time on this.

Internal ticket: DTRSD-13008

Signed-off-by: Pierre De Paepe <pierre.de-paepe@ovhcloud.com>

### CODE REVIEW

File: packages/manager/modules/server-sidebar/src/order.constants.js [hunk 1]
```diff
@@ -252,6 +252,46 @@ export const ORDER_URLS = {
       PL: 'https://www.ovh.com/manager/sunrise/index.html#/csp2',
       PT: 'https://www.ovh.com/manager/sunrise/index.html#/csp2',
     },
+    dbaas_logs: {
+      CA:
```
Comment (marie-j) [inline packages/manager/modules/server-sidebar/src/order.constants.js:256]: Order constants are partitioned by IS : https://github.com/ovh/manager/blob/fb660dd670c274697e868830ceb8d4085f06e481/packages/manager/modules/server-sidebar/src/order.constants.js#L4 and https://github.com/ovh/manager/blob/fb660dd670c274697e868830ceb8d4085f06e481/packages/manager/modules/server-sidebar/src/order.constants.js#L296

So `CA,QC, WE,WS` (I notice no ASIA, AU and SG) should be declared in the CA part instead maybe

File: packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.html [hunk 2]
```diff
@@ -155,10 +155,7 @@
                                 <a
                                     data-ng-href="{{ ctrl.accountDetails.data.elasticSearchApiUrl }}"
                                     class="oui-tile__button oui-button oui-link_icon oui-button_link"
-                                    data-ng-class="{
-                                        'oui-link_disabled': !(ctrl.account.data.total.maxNbIndex > 0 || ctrl.account.data.total.maxNbAlias > 0),
-                                        'oui-button_full-width': (ctrl.account.data.total.maxNbIndex > 0 || ctrl.account.data.total.maxNbAlias > 0)
-                                   }"
+                                    data-ng-class="oui-button_full-width"
```
Comment (marie-j) [inline packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.html:158]: Those changes doesn't look related to the order button, maybe they can be applied in a separate commit

File: packages/manager/modules/server-sidebar/src/order.constants.js (hunk 1 above)
Comment (pdepaepe) [reply packages/manager/modules/server-sidebar/src/order.constants.js]: Fixed, thanks.

File: packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.html (hunk 2 above)
Comment (pdepaepe) [reply packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.html]: Ok, will do an other PR
